presto-plugin-toolkit/pom.xml

```diff
@@ -121,5 +131,11 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
```

Comment:
I don't want to take a dependency on mockito. In Presto we hand roll our own mocks